src/mainvolume/module-meego-mainvolume.c

```diff
@@ -898,6 +958,11 @@ static pa_dbus_signal_info mainvolume_signals[MAINVOLUME_SIGNAL_MAX] = {
         .name = "CallStatus",
         .arguments = call_state_args,
         .n_arguments = 1
+    },
+    [MAINVOLUME_SIGNAL_MEDIA_STATE] = {
+        .name = "MediaState",
```

Comment:
Ponder if this should indicate changed state in the name, there's e.g. "StepsUpdated", but then again "CallStatus", a bit inconsistent. Doesn't matter much, though.